main.cpp

```diff
@@ -307,7 +281,7 @@ std::string Student::skipClass(std::string lesson1)
 
 struct Teacher
 {
-    Teacher();
+    Teacher(): age {45}, heightInCentimeters {175.2f}, compensation {90000},highestDegreeHeld {"PhD"}, position {"Senior Professor"} {}
```

Comment:
check the spacing around your `,`.  it can be helpful to put each member being initialized on its own line.
whatever it takes the make the code as readable as possible...
- [ ] revise